src/components/progress-bar/__test__/index.test.js

```diff
@@ -0,0 +1,69 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import ReactDOM from 'react-dom';
+import { render } from '@testing-library/react';
+import '@testing-library/jest-dom';
+
+/**
+ * Internal dependencies
+ */
+import ProgressBar from './../index';
+
+describe( 'The ProgressBar component', () => {
+	it( 'renders without crashing', () => {
+		const div = document.createElement( 'div' );
+		ReactDOM.render( <ProgressBar></ProgressBar>, div );
+	} );
+
+	it( 'shows the classnames of the outer div correctly', () => {
+		const { getByTestId } = render( <ProgressBar></ProgressBar> );
+		expect( getByTestId( 'progress-bar-outer' ) ).toHaveClass(
+			'wc-free-shipping-progress-bar__outer'
+		);
+	} );
```

Comment:
Using a testId here is not needed, you can just return the `container` from the render call, and have it in the expect check.

Reply:
Also, this runs by the assumption that the class name won't change, is this what you're trying to test here?

Reply:
Thanks for pointing this out. I replaced `getByTestId()` with `document.querySelector()` to keep the code lean.